hawkular-inventory-dist/pom.xml

```diff
@@ -138,47 +138,20 @@
 
     <dependency>
       <groupId>org.umlg</groupId>
-      <artifactId>sqlg-hsqldb</artifactId>
+      <artifactId>sqlg-hsqldb-dialect</artifactId>
       <version>${version.org.umlg}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.apache.tinkerpop</groupId>
-          <artifactId>gremlin-groovy</artifactId>
-        </exclusion>
-      </exclusions>
     </dependency>
 
     <dependency>
       <groupId>org.umlg</groupId>
-      <artifactId>sqlg-postgres</artifactId>
+      <artifactId>sqlg-postgres-dialect</artifactId>
       <version>${version.org.umlg}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.apache.tinkerpop</groupId>
-          <artifactId>gremlin-groovy</artifactId>
-        </exclusion>
-      </exclusions>
     </dependency>
 
     <dependency>
       <groupId>org.umlg</groupId>
-      <artifactId>sqlg-h2</artifactId>
+      <artifactId>sqlg-h2-dialect</artifactId>
```

Comment:
Should we completely remove artifacts for the in-memory store we are not using (i.e. if we use hsqldb, then remove h2-support and vice versa) ?